ARM.CMSIS.pdsc

```diff
@@ -2174,16 +1308,290 @@ and 8-bit Java bytecodes in Jazelle state.
       <require Cclass="Device" Cgroup="Startup"/>
     </condition>
 
+    <condition id="RTOS RTX ARM ARMv6-M">
+        <description>Arm Compiler for Armv6-M architecture</description>
+        <require condition="ARMCC"/>
+        <require condition="ARMv6-M Device"/>
+        <require Dendian="little"/>
+    </condition>
+    <condition id="RTOS RTX ARM ARMv6-M BE">
+        <description>Arm Compiler for Armv6-M architecture (big endian)</description>
+        <require condition="ARMCC"/>
+        <require condition="ARMv6-M Device"/>
+        <require Dendian="big"/>
+    </condition>
+    <condition id="RTOS RTX ARM ARMv7-M NOFP">
+        <description>Arm Compiler for Armv7-M architecture without FPU</description>
+        <require condition="ARMCC"/>
+        <require condition="ARMv7-M Device"/>
+        <require Dendian="little"/>
+        <require Dfpu="NO_FPU"/>
+    </condition>
+    <condition id="RTOS RTX ARM ARMv7-M BE NOFP">
```

Comment:
For me it would be more readable if BE/LE would be at the end (architecture and FP belong together).
"ARMCC ARMv7-M NOFP BE"